.github/workflows/build.yml

```
@@ -14,49 +28,107 @@ jobs:
+              - 'backend/**'
+              - 'frontend/**'
+              - 'build/**'
+              - 'go.mod'
+              - 'go.sum'
+              - 'package.json'
+              - 'Taskfile*.yml'
+              - '.github/workflows/build.yml'
+
+      - name: Check if build should run
+        id: should-build
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:28: Double quote to prevent globbing and word splitting

(shellcheck)